### PR TITLE
8327757: Convert javax/swing/JSlider/6524424/bug6524424.java applet to main

### PR DESCRIPTION
**Pull request by prsadhuk**
Conversion of manual applet test to main based using PassFailJFrame manual framework

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8327757](https://bugs.openjdk.org/browse/JDK-8327757): Convert javax/swing/JSlider/6524424/bug6524424.java applet to main (**Bug** - P4)

### Reviewers
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - Committer)
 * [Jayathirth D V](https://openjdk.org/census#jdv) (@jayathirthrao - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18263/head:pull/18263` \
`$ git checkout pull/18263`

Update a local copy of the PR: \
`$ git checkout pull/18263` \
`$ git pull https://git.openjdk.org/jdk.git pull/18263/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18263`

View PR using the GUI difftool: \
`$ git pr show -t 18263`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18263.diff">https://git.openjdk.org/jdk/pull/18263.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18263#issuecomment-1993740820)

### CODE REVIEW

File: test/jdk/javax/swing/JSlider/bug6524424.java [hunk 1]
```diff
@@ -21,28 +21,56 @@
  * questions.
  */
 
-/* @test
+/*
+ * @test
  * @bug 6524424
  * @requires (os.family == "windows")
  * @summary JSlider Clicking In Tracks Behavior Inconsistent For Different Tick Spacings
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JSlider/bug6524424.java:28]: Minor suggestion: All words need not to be start with Capital letter.
```suggestion
 * @summary JSlider clicking in tracks behavior inconsistent for different tick spacings
```

File: test/jdk/javax/swing/JSlider/bug6524424.java [hunk 2]
```diff
@@ -52,15 +80,7 @@ public static void main(String[] args) {
         frame.setContentPane(panel);
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JSlider/bug6524424.java:80]: `JFrame frame = new JFrame();`

Please add suitable frame title.

File: test/jdk/javax/swing/JSlider/bug6524424.java [hunk 3]
```diff
@@ -52,15 +80,7 @@ public static void main(String[] args) {
...
         frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JSlider/bug6524424.java:81]: `frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE); ` redundant can be removed.

**Review by kumarabhi006 — APPROVED** [review]

**Review by jayathirthrao — APPROVED** [review]

**Pull request closed without merging**